micropython/drivers/led/neopixel/neopixel.py

```diff
@@ -45,6 +58,13 @@ def fill(self, v):
                 b[j] = c
                 j += bpp
 
+    def set_brightness(self, b: float):
```

Comment:
Rename to `brightness` (now that that name isn't used for the field). It's more in line with what we do elsewhere (e.g. `pin.value()`).

Also I think, similarly, it should probably return the current brightness too, e.g.

```py
    def brightness(self, brightness):
        if brightness is not None:
            self.b = min(max(brightness, 0), 1)
            # This may look odd but because __getitem__ and __setitem__ handle all the
            # brightness logic already, we can offload the work to those methods.
            for i in range(self.n):
                self[i] = self[i]
        return self.b
```

which is +10 bytes, but I think useful for people to write e.g. `strip.brightness(strip.brightness() + 0.1)`.